games-rpg/flare/flare-1.14.recipe

```diff
@@ -90,11 +90,13 @@ BUILD()
 {
 	sdl2dir=$(finddir B_SYSTEM_HEADERS_DIRECTORY)${secondaryArchSubDir}/SDL2
 
-	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release -Wno-dev \
+	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_POLICY_VERSION_MINIMUM=3.5 \
 		-DSDL2_INCLUDE_DIR=$sdl2dir \
 		-DSDL2_IMAGE_INCLUDE_DIR=$sdl2dir \
 		-DSDL2_MIXER_INCLUDE_DIR=$sdl2dir \
-		-DSDL2_TTF_INCLUDE_DIR=$sdl2dir
+		-DSDL2_TTF_INCLUDE_DIR=$sdl2dir \
+		-Wno-devx
```

Comment:
Typo with the 'x' at the end?

Reply:
LOL, indeed, guess a bit too fast in typing :) (luckaly didn't harm the build on buildmaster